addon/helpers/contains.js

```diff
@@ -4,7 +4,7 @@ import { isEmberArray } from 'ember-array/utils';
 import createNeedleHaystackHelper from '../-private/create-needle-haystack-helper';
 
 function _contains(needle, haystack) {
-  return emberArray(haystack).contains(needle);
+  return emberArray(haystack).includes(needle);
```

Comment:
Might want to still use `contains`, isn't `includes` new? This will break old versions of Ember.